.devcontainer/setup

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+sudo apt-get update -y
+sudo apt-get install -y libturbojpeg0
+
```

Comment:
This was added because I was tired of seeing the Home Assistant error for PyTurboJPEG logged on every run